src/models/user.js

```diff
@@ -1,28 +1,41 @@
 export default (sequelize, DataTypes) => {
-  const User = sequelize.define('User', {
-    username: {
-      allowNull: false,
-      type: DataTypes.STRING,
-      unique: true
-    },
-    password: {
-      allowNull: false,
-      type: DataTypes.STRING
-    },
-    lastLogin: {
-      allowNull: true,
-      type: DataTypes.DATE,
-      defaultValue: sequelize.fn('NOW')
-    },
-    email: {
-      allowNull: false,
-      type: DataTypes.STRING,
-      unique: true
-    },
-    phoneNumber: {
-      allowNull: false,
-      type: DataTypes.STRING
+  const User = sequelize.define(
+    "User",
```

Comment:
double quotes, do you have eslint installed on your vscode?

Reply:
I actually do. Thanks. This has been resolved.